api/envoy/service/load_stats/v2/lrs.proto

```diff
@@ -66,7 +66,13 @@ message LoadStatsRequest {
 // [#not-implemented-hide:] Not configuration. TBD how to doc proto APIs.
 message LoadStatsResponse {
   // Clusters to report stats for.
+  // Not populated if send_all_clusters is true.
   repeated string clusters = 1 [(validate.rules).repeated = {min_items: 1}];
+  // If true, the client should send all clusters it knows about.
+  // The server may only specify this if the client advertises the
```

Comment:
I think it's safe to set this regardless of the capability, you just might not get back the expected result.

Reply:
Reworded to indicate that clients will not honor the field unless they advertise that capability.